common/test/acceptance/pages/lms/discussion.py

```diff
@@ -354,12 +354,11 @@ def close_open_thread(self):
         with self.thread_page._secondary_action_menu_open(".forum-thread-main-wrapper"):
             self._find_within(".forum-thread-main-wrapper .action-close").first.click()
 
-    @wait_for_js
-    def is_window_on_top(self):
+    def is_focus_changed(self, selector):
```

Comment:
I think this method should be renamed to something a bit more accurate. Specifically that there is focus or there is not focus.